### ISSUE TITLE
Change one color without changing the other defaults

### ISSUE THREAD

**Issue by taylor-scott** [mozilla/FirefoxColor#991]
I would like to change the background color in the "custom colors" tab. I only want to change the background color, and I do not want any of the other default colors that are loaded in this tab. When I click "background color" and change it, it automatically switches to all of the other colors (pink tab highlight, purple search text, etc.) which I do not want. How can I change a single color while not updating everything else? Is there a list of "firefox default" values that I can use to fix all of the other colors?

![image](https://user-images.githubusercontent.com/3229203/129392210-5c906353-21c9-4381-810c-1320d7e3044c.png)

**Comment by wheelerlaw** [comment]
Yeah, this issue makes this add-on functionally unusable if you just want to tweak the colors of the system theme.

**Comment by Brasolis** [comment]
Wow I see this is still an issue 4 years later. I was just looking up how to change a single thing without making everything else awful. Crazy how this isn't an option.
